src/util/time.rs

```diff
@@ -1,13 +0,0 @@
-//! Util time conversion functions
-
-use std::time::Duration;
-
-///Converts a Duration to the time in seconds.
-pub fn duration_to_secs(duration: Duration) -> f32 {
```

Comment:
I like this function, seems like it didn't get added back.  Can we put it back somewhere?

Reply:
Ok, it wasn't used anywhere was all.

Reply:
Mostly a visibility problem :/ it should be used.

Reply:
A grep on the project showed no uses. The other method was used in one place only, so I moved it there. But I can return this module in the core crate no worries. Will be tomorrow though.

Reply:
What I meant was it works great in places like this: https://github.com/amethyst/amethyst/blob/develop/examples/04_pong/systems/move_balls.rs#L18 it just wasn't used there because we all forgot it existed.

Reply:
(Last night while working on my prototype I was wanting something like this, I totally forgot we already had it)

Reply:
Oki, will remedy tomorrow.

Reply:
Will move it to the timing module